src/core/rfc822/MCMessageBuilder.h

```diff
@@ -40,7 +40,10 @@ namespace mailcore {
         
         virtual Data * data();
         virtual Data * dataForEncryption();
-        
+
+        // Store builded message to file. Return false in case of failure.
+        virtual bool storeToFile(String * filename);
```

Comment:
Could you rename all storeToFile() to writeToFile() and return a ErrorCode instead?
(to be consistent with Data::writeToFile()).

Thanks!